docs/src/docs/reports/timings-view.adoc

```diff
@@ -0,0 +1,29 @@
+= Timings View
+
+== Tests tab pane
+All executed test cases are included in a bar chart, grouped in ranges by their execution time.
+By hovering over a bar the methods within this range are listed in a tooltip.
+
+It is possible to adjust the number of method ranges by selecting a value of 5, 10, 15 or 20 in the corresponding drop-down menu.
+This way you can adjust the resolution of the chart.
+
+You can also search for a specific method by entering the name of the test case in the searchbar.
+The corresponding bar is then highlighted.
+
+By default, only the test methods themselves are included. However, it is possible to incorporate the configuration methods by toggling the switch located in the top right corner of the view.
+
+image::report-ng-timings-view_tests.png[align="center", alt="Tests"]
+
+== Sessions tab pane
+In the scatter chart you can see every session that was started in the test execution.
```

Comment:
If a base URL is configured, also the load duration and the timestamp of a base URL is shown. In that case the two dots are "siblings": The base URL is always loaded at session startup. Keep in min, that setup a bse URL is optional.

If both dots exists the hover effect shows which base URL dot belongs to which session dot.